foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -728,11 +730,11 @@ msgstr ""
 "Sie können sich mit dem Turris-Omnia-Router am Turris-Projekt beteiligen. "
 "Das Turris-Projekt ist ein gemeinnütziges Forschungsprojekt gegründet von "
 "der Organisation CZ.NIC, welche die .CZ Domäne verwaltet. Falls Sie dem "
-"Projekt beitreten, wird sich Ihr Router als eine Sonde benehmen, welche den "
-"Datenfluss zwischen ihrem Heim und dem Internet analysiert, um arglistige "
-"Datenströme auszufinden. Wenn solcher Datenstrom gefunden ist, dann wird die "
-"Turris-Zentrale notifiziert, welche die Bedrohungsstufe mittels "
-"Datenvergleich festlegen kann. Wenn der Datenstrom als eine Attacke "
+"Projekt beitreten, wird sich Ihr Router als eine Sonde verhalten, welche den "
+"Datenfluss zwischen ihrem Heimnetzwerk und dem Internet analysiert, um "
+"böseartige Datenströme zu finden. Wenn solcher Datenstrom gefunden ist, dann "
```

Comment:
If there is someone to blame then it's definitely me as I'm the person who's responsible for it. So many mistakes between 'Sie' and 'ihr', shame on me. This is something (and not only that) I should focus on next time.

Could you please correct all the small typos and mistakes we both agreed on? My colleague will merge all the changes as soon as you will fix them.

Many thanks for fixing my errors.